source-google-sheets-native/source_google_sheets_native/api.py

```diff
@@ -105,7 +102,7 @@ def convert_row(
             else:
                 d[headers[ind]] = nv
 
-    return Row(_meta=Row.Meta(op="u", row_id=id), **d)
```

Comment:
Setting the `op` and `row_id` here never did anything since they are set by the CDK.